packages/speech/src/index.js

```diff
@@ -87,6 +88,9 @@ function Speech(options) {
   };
 
   common.GrpcService.call(this, config, options);
+  this.api = {
```

Comment:
I'm cool with that. Small tweak, maybe `this.api.speech` since `this.api.speechApi` is somewhat redundant.